### PR TITLE
some nits for advanced audit document

### PR DESCRIPTION
**Pull request by CaoShuFeng**
> NOTE: Please check the “Allow edits from maintainers” box (see image below) to  
> [allow reviewers to fix problems](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) on your patch and speed up the review process.
>
> Please delete this note before submitting the pull request.

Allow edits from maintainers checkbox

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/4259)

### CODE REVIEW

File: docs/tasks/debug-application-cluster/audit.md [hunk 1]
```diff
@@ -84,9 +84,9 @@ webhooks.
 
 The structure of audit events changes when enabling the `AdvancedAuditing` feature
 flag. This includes some cleanups, such as the `method` reflecting the verb evaluated
-by the [authorization layer](/docs/admin/authorization/) instead of the HTTP verb
-("create" instead of "POST"). Also, instead of always generating two events per
-request, events are recorded with an associated "stage." The known stages are:
+by the [authorization layer](/docs/admin/authorization/) instead of the [HTTP verb] (docs/admin/authorization/#determine-the-request-verb).
```
Comment (sttts) [inline docs/tasks/debug-application-cluster/audit.md]: nit: no space before `(doc`

**Comment by sttts** [comment]
One nit. Otherwise, lgtm.

File: docs/tasks/debug-application-cluster/audit.md (hunk 1 above)
Comment (CaoShuFeng) [inline docs/tasks/debug-application-cluster/audit.md]: Done.
Thanks.

**Pull request merged**